### PR TITLE
Rename JSON keys of the access log

### PR DESCRIPTION
**Pull request by renuka-fernando**
### Purpose
$subject

### Automation tests
 - Unit tests added: No
 - Integration tests added:No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

#### Sample Logs

```json
{"RespTxDur":0,"ReqPath":"/testkey","RespDur":165,"RespCodeDtls":"via_upstream","XFF":null,"BytesSent":762,"ReqID":"d058d9cb-861c-4849-ab4f-fea76f2f5357","Host":"enforcer","RespFlag":"-","UpstrmSvcTime":null,"Method":"POST","GwPath":null,"Prot":"HTTP/1.1","RName":"system#/testkey","Time":"2023-07-10T08:43:04.059Z","BytesRecv":15,"Dur":165,"UpstrmHost":"172.18.0.4:9001","ReqTxDur":101,"RespCode":200,"GwHost":null,"UA":"curl/7.79.1","ReqDur":2}

{"RespFlag":"-","RespCodeDtls":"via_upstream","RespCode":200,"Method":"GET","ReqTxDur":1010,"GwPath":"/v2/pet/findByStatus?status=pending","ReqDur":0,"ReqPath":"/anything/pet/findByStatus?status=pending","UA":"curl/7.79.1","Prot":"HTTP/1.1","Dur":2115,"XFF":null,"BytesRecv":0,"RespDur":2115,"Host":"httpbin.org","Time":"2023-07-10T08:43:12.869Z","GwHost":"localhost:9095","UpstrmSvcTime":null,"ReqID":"03bdc3c0-81ad-42fc-9de3-8539bce1cb50","UpstrmHost":"54.204.94.184:443","BytesSent":459,"RName":"648957e7a675c35145393369","RespTxDur":0}
```

### CODE REVIEW

**Comment by malinthaprasan** [comment]
Shall we define keys using "camel" format?

**Review by malinthaprasan — APPROVED** [review]

**Review by VirajSalaka — APPROVED** [review]

File: resources/conf/log_config.toml [hunk 1]
```diff
@@ -43,26 +43,26 @@ logType = "text" # LogTypes can be "text" or "json"
...
-resTxDu = "%RESPONSE_TX_DURATION%"
-reqDu = "%REQUEST_DURATION%"
-resDu = "%RESPONSE_DURATION%"
+xff = "%REQ(X-FORWARDED-FOR)%"
+upstrmHost = "%UPSTREAM_HOST%"
+bytesRecv = "%BYTES_RECEIVED%"
+bytesSent = "%BYTES_SENT%"
+dur = "%DURATION%"
+upstrmSvcTime = "%RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)%"
+reqTxDur = "%REQUEST_TX_DURATION%"
```
Comment (slahirucd7) [inline resources/conf/log_config.toml:65]: Is it possible to clarify more the term `Tx` value ?

Comment (renuka-fernando) [reply resources/conf/log_config.toml:65]: I guess it is a common abbreviation used for transmission and for transactions.

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : dev-deployment-v2 : 20230710.29

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20230710.29

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20230710.29

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : stage-deployment-v2 : 20230711.2

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : stage-deployment-v2 : 20230711.2

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20230711.2

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20230711.2

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20230711.2

**Pull request merged**